src/Components/FormCreation/GetPreviewLinkDialog.js

```diff
@@ -0,0 +1,259 @@
+    display: block;
+    align-items: center;
+    letter-spacing: 0.15px;
+  }
+  input {
+    height: 24px;
+    left: 0%;
+    right: 0%;
+    width: 100%;
+    border: 0px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 21px;
+    display: block;
+    align-items: center;
+    letter-spacing: 0.25px;
+  }
+  input:focus {
+    outline: none;
+  }
+`;
+
+const ButtonWrapper = styled.div`
+  position: absolute;
+  bottom: 18px;
+  right: 19px;
+  button {
+    text-transform: none;
+    font-weight: 500;
+    font-size: 14px;
+    text-transform: uppercase;
+  }
+`;
+
+function GetPreviewLinkDialog({ close, link, copyLinkToClipboard }) {
+  const dialogRef = React.createRef();
+
+  useEffect(() => {
+    function detectEscape(event) {
+      if (event.keyCode === 27) {
+        close();
+      }
+    }
+
+    function detectOutsideClicks(event) {
+      if (dialogRef.current && !dialogRef.current.contains(event.target)) {
+        close();
+      }
+    }
+
+    if (dialogRef.current) {
+      window.addEventListener("keydown", detectEscape);
+      window.addEventListener("mousedown", detectOutsideClicks);
+    }
+  }, [close, dialogRef]);
+
+  const handleFocus = (event) => {
+    copyLinkToClipboard();
+    event.preventDefault();
+    event.target.select();
+  };
+
+  return (
+    <Dialog
+      ref={dialogRef}
+      showClose={true}
+      paddingHorizontal={28}
+      paddingVertical={28}
+      style={{
+        width: "788px",
+        height: "312px",
+        boxShadow:
+          "0px 8px 10px rgba(0, 0, 0, 0.14), 0px 3px 14px rgba(0, 0, 0, 0.12), 0px 5px 5px rgba(0, 0, 0, 0.2)",
+        borderRadius: "4px",
+        padding: "18px"
+      }}
+    >
+      <Header
+        style={{ lineHeight: "36px", fontSize: "24px", fontWeight: "400" }}
+      >
+        Get Preview Link
+        <IconButton
+          onClick={close}
+          style={{
+            display: "inline-block",
+            marginLeft: "auto"
+          }}
+          size="small"
+        >
+          <Close />
+        </IconButton>
+      </Header>
+      <WarningMessage>
+        <svg
+          style={{
+            position: "absolute",
+            left: "5.83%",
+            right: "91.26%",
+            top: "116px"
+          }}
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M12 2C8.14 2 5 5.14 5 9C5 11.38 6.19 13.47 8 14.74V17C8 17.55 8.45 18 9 18H15C15.55 18 16 17.55 16 17V14.74C17.81 13.47 19 11.38 19 9C19 5.14 15.86 2 12 2ZM9 21C9 21.55 9.45 22 10 22H14C14.55 22 15 21.55 15 21V20H9V21ZM14 13.7L14.85 13.1C16.2 12.16 17 10.63 17 9C17 6.24 14.76 4 12 4C9.24 4 7 6.24 7 9C7 10.63 7.8 12.16 9.15 13.1L10 13.7V16H14V13.7Z"
+            fill="black"
+            fill-opacity="0.54"
```

Comment:
Should be fillRule, clipRule, and fillOpacity. See travis CI failure.

Reply:
Fixed. Also all comments during demo were already fixed.